Code/GraphMol/FileParsers/ProximityBonds.h

```diff
@@ -12,7 +12,10 @@
 #include <GraphMol/RWMol.h>
 
 namespace RDKit {
-void ConnectTheDots(RWMol *mol);
+static unsigned int ctdIGNORE_H_H_CONTACTS = 0x1;
+static unsigned int ctdALL_FLAGS = 0xFFFFFFFF;
+
+void ConnectTheDots(RWMol *mol, unsigned int flags = 0);

```

Comment:
Why do you introduce a flag for the H-H case? Do you see a situation where this might be set to False?